bin/s3rver.js

```diff
@@ -91,7 +91,20 @@ program.on('--help', () => {
 program.action(async command => {
   const { configureBucket, ...opts } = command.opts();
   opts.configureBuckets = configureBucket;
-  const { address, port } = await new S3rver(opts).run();
+
+  const server = new S3rver(opts);
+  const { address, port } = await server.run();
+
+  process.on('SIGINT', async function onSigint() {
```

Comment:
From [Node docs](https://nodejs.org/api/process.html#process_signal_events):

> 'SIGTERM' and 'SIGINT' have default handlers on non-Windows platforms that reset the terminal mode before exiting with code 128 + signal number. **If one of these signals has a listener installed, its default behavior will be removed (Node.js will no longer exit).**

I think this means you must call `process.exit` inside these handlers or else the `node` process won't be terminated via `SIGINT` or `SIGTERM`, you would have to send a `SIGKILL` to terminate the process.